app/db/migrations/versions/2a0a7fdef0f6_add_users_to_db.py

```diff
@@ -1,40 +1,34 @@
 """Add users to db
 
 Revision ID: 2a0a7fdef0f6
-Revises: 806702adc8af
+Revises:
 Create Date: 2024-08-19 12:17:26.026339
 
 """
 
-from typing import Sequence, Union
-
 from alembic import op
 import sqlalchemy as sa
 import sqlmodel
 
 
 # revision identifiers, used by Alembic.
 revision: str = "2a0a7fdef0f6"
-down_revision: Union[str, None] = "806702adc8af"
-branch_labels: Union[str, Sequence[str], None] = None
-depends_on: Union[str, Sequence[str], None] = None
+down_revision = None
+branch_labels = None
+depends_on = None
 
 
 def upgrade() -> None:
-    # ### commands auto generated by Alembic - please adjust! ###
     op.create_table(
         "users",
         sa.Column("username", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
         sa.Column("hashed_password", sqlmodel.sql.sqltypes.AutoString(), nullable=True),
```

Comment:
Can nullable be set to False please so a user can't be added with a null password.